packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/ShowCollaborator.spec.js

```diff
@@ -273,6 +273,39 @@ describe('Collaborator component', () => {
         })
       })
     })
+    describe('role', () => {
+      it.each([
+        { name: 'viewer', expectedText: 'Viewer' },
+        { name: 'editor', expectedText: 'Editor' },
+        { name: 'advancedRole', expectedText: 'Advanced permissions' },
+        { name: 'owner', expectedText: 'Owner' },
+        { name: 'resharer', expectedText: 'Resharer' },
+        { name: 'invalidRole', expectedText: 'Unknown Role' }
+      ])('shows the role label', cases => {
+        const wrapper = createWrapper({
+          role: { name: cases.name }
+        })
+        expect(wrapper.find('.files-collaborators-collaborator-role').text()).toEqual(
```

Comment:
you can reuse this selector using as a variable

Reply:
@kiranparajuli589 done, please review again